hystrix-contrib/hystrix-metrics-event-stream/src/main/java/com/netflix/hystrix/contrib/metrics/eventstream/HystrixMetricsStreamServlet.java

```diff
@@ -135,7 +134,17 @@ private void handleRequest(HttpServletRequest request, HttpServletResponse respo
                 response.setHeader("Cache-Control", "no-cache, no-store, max-age=0, must-revalidate");
                 response.setHeader("Pragma", "no-cache");
 
-                MetricJsonListener jsonListener = new MetricJsonListener();
+                int queueSize = defaultMetricListenerQueueSize.get();
+                try {
+                    String q = request.getParameter("queueSize");
```

Comment:
IMO, the velocity of metrics is independent of who is consuming the metrics.  What problem do you intend to solve by allowing the request to set the queue size?